src/pages/Search/SearchPage.tsx

```diff
@@ -0,0 +1,23 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React from 'react';
+import ScreenWrapper from '@components/ScreenWrapper';
+import type {CentralPaneNavigatorParamList} from '@libs/Navigation/types';
+import type SCREENS from '@src/SCREENS';
+import SearchResults from './SearchResults';
+import useCustomBackHandler from './useCustomBackHandler';
+
+type SearchPageProps = StackScreenProps<CentralPaneNavigatorParamList, typeof SCREENS.SEARCH.CENTRAL_PANE>;
+
+function SearchPage({route}: SearchPageProps) {
+    useCustomBackHandler();
+
+    return (
+        <ScreenWrapper testID={SearchPage.displayName}>
```

Comment:
When this page was created, the `offline pattern` was omitted which led to this issue:
- https://github.com/Expensify/App/issues/43556